llvm/test/TableGen/ConcatenatedSubregs.td

```diff
@@ -90,16 +90,16 @@ def TestTarget : Target;
 // CHECK-LABEL: RegisterClass DRegs:
 
 // CHECK-LABEL: SubRegIndex ssub1:
-// CHECK:        Offset, Size: 16, 16
+// CHECK:        Offset, Size: { Default:16, 16 }
```

Comment:
Perhaps this should use extra {} to group the pair? One could be forgiven for thinking the Default only applies to the first argument, and it'll look very strange once you add in more HwMode's.

Reply:
I split the Offset and Size on different lines.